client/src/components/ManageAccount/ManageAccountTables.js

```diff
@@ -183,14 +184,69 @@ export const Licenses = () => {
   );
 };
 
-export const ThirdPartyApps = () => {
+export const IntegrationCell = ({ cell: { value } }) => {
+  const { activated, label } = value;
+  switch (label) {
+    case 'Google Drive':
+      return <GoogleDriveIntegrationCell activated={activated} />;
+    default:
+      return <></>;
+  }
+};
+IntegrationCell.propTypes = {
+  cell: shape({
+    value: shape({
+      activated: bool.isRequired,
+      label: string.isRequired
+    })
+  }).isRequired
+};
+
+export const GoogleDriveIntegrationCell = ({ activated }) => {
```

Comment:
I think this needs to be another configurable. Given we use `_PORTAL_DATAFILES_STORAGE_SYSTEMS` to determine what is available in the Data Files area, would it make sense to search this object here? And perhaps change the `scheme` from `private` to `integration`, like so:
```
_PORTAL_DATAFILES_STORAGE_SYSTEMS = [
    {
        'name': 'Community Data',
        'system': _AGAVE_COMMUNITY_DATA_SYSTEM,
        'scheme': 'community',
        'api': 'tapis',
        'icon': None
    },
    {
        'name': 'Shared Workspaces',
        'scheme': 'projects',
        'api': 'tapis',
        'icon': None
    },
    {
        'name': 'Google Drive',
        'scheme': 'integration',
        'api': 'googledrive',
        'icon': None
    }
]
```
This would also allow the disabling of utility buttons based on `scheme === 'integration` instead of `api`

Reply:
On the Manage Profile side the configurability is already there. If the google drive app isn't installed in the settings then the table is empty.

Reply:
👍